extensions/plugins/system/redcore/redcore.php

```diff
@@ -90,6 +90,11 @@ public function onAfterInitialise()
 						$data = RApi::getPostedData();
 						$dataGet = $input->get->getArray();
 
+						if (!empty($data['language']) && empty($dataGet['language']))
```

Comment:
I actually tried to do this by just manually setting the language to the app input, but that was at the beginning before I found the issue in RDatabaseSqlparserSqltranslation::buildTranslationQuery and RApiSoapOperationOperation::setLanguage

So I guess we can resolve this and the other chagne inside the RApiSoapOperationOperation after the data has been sent to the readItem method